definitions/mobile.d.ts

```diff
@@ -38,6 +38,10 @@ declare module Mobile {
 		startService(serviceName: string): number;
 	}
 
+	interface ILogcatHelper {
+		start(deviceIdentifier: string, adbPath: string): any;
```

Comment:
Hm, it's a little bit strange to have `adbPath` as parameter. `start` function can resolve it internal. We can get it from `helpers.getPathToAdb()`